CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Add `--exclude-pods` option to `pod update` to allow excluding specific pods from update
```

Comment:
nit of nits, add two empty spaces after the first message

Reply:
Oh I see, for the line break, will do 👍